Automattic/wp-calypso
# Stats: Replace usages of the "Views" icon with the eye component

Related to #71933 

## Proposed Changes

* Replace the `Views` icon usage with the `eye` component.
* Replace the importing for the icon eye in the `StatsEmailTopRow` and delete the duplicated `eye` component.

## Testing Instructions



* Spin up this change with the Calypso Live link.
* Ensure all Views-related icons in [the file list](https://github.com/Automattic/wp-calypso/issues/71933) work as previously.

## Pre-merge Checklist
